pyproject.toml

```diff
@@ -142,6 +142,7 @@ files = [
 ]
 exclude = [
     "dbt/adapters/events/adapter_types_pb2.py",
+    "dbt-tests-adapter/dbt/__init__.py",  # overlaps with `dbt/__init__.py` as expected for namespaces
```

Comment:
Since we split the namespace package, this shows up twice for `mypy`, which means the same path goes to two files. This is expected and works itself out once the namespace packages are combined.